.github/workflows/acceptance_tests_base.yml

```
@@ -213,6 +213,19 @@ jobs:
       - name: Run acceptance tests
         if: ${{ !inputs.skip_tests && needs.filters.outputs.require_acceptance_tests == 'true' }}
         run: ./testsuite/podman_runner/${{ inputs.tests }}
+      
+      - name: Upload Cucumber JSON Reports
+        uses: actions/upload-artifact@v4
```

For security reasons, the team decided to use commit hashes instead of tags in all our GH actions.
```suggestion
        uses: actions/upload-artifact@4cec3d8aa04e39d1a68397de0c4cd6fb9dce8ec1 #v4.6.1
```